x/staking/legacy/v0_36/types.go

```diff
@@ -72,3 +87,49 @@ func NewGenesisState(
 		Exported:             exported,
 	}
 }
+
+func (v Validator) MarshalJSON() ([]byte, error) {
```

Comment:
Note, the `Validator` type in staking implements custom JSON marshaling. Hence, we have to copy that logic here.